src/VisualStudio/Core/Def/ProjectSystem/OpenTextBufferProvider.cs

```diff
@@ -57,12 +57,12 @@ public OpenTextBufferProvider(
         [Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider,
         IAsynchronousOperationListenerProvider listenerProvider)
     {
+        // This ctor does RDT operations, and thus requires the UI thread.
```

Comment:
Maybe make some comment here for whoever gets to delete this, to add some comment back over the RDT GetService so people know what's allowed again. And I guess update the version number in the comment accordingly. :smile:

Reply:
Comments added.